extension/docs/ManualCapture.md

```diff
@@ -0,0 +1,88 @@
+**Please see [Integration Guide](WebComponentsIntegrationGuide.md) first before continuing with this document.**
+
+## Manual Capture
+By default, payments are captured immediately after authorisation. For payment methods that support separate authorisation and capture, you also have the option to capture the payment later, for example only after the goods have been shipped. This allows you to cancel the payment in case of any issues with the shipment. 
+
+### Make an API call to capture a payment:
+
+```json
+{
+   "modificationAmount":{
+      "value":500,
+      "currency":"EUR"
+   },
+   "originalReference":"8313547924770610",
+   "reference": "YOUR_UNIQUE_REFERENCE"
+}
+```
+
+An [update action](https://docs.commercetools.com/http-api-projects-payments#set-customfield) to set `manualCaptureRequest` custom field.
+
+``` json
+{
+  "action": "setCustomField",
+  "name": "manualCaptureRequest",
+  "value": "{\"modificationAmount\":{\"value\":500,\"currency\":\"EUR\"},\"originalReference\":\"8313547924770610\",\"reference\":\"YOUR_UNIQUE_REFERENCE\"}"
+}
+```
+
+The `manualCaptureResponse` contains a PSP reference associated with this manual capture request.
+
+``` json
+{
+   "pspReference":"8825408195409505",
+   "response":"[capture-received]"
+} 
+```
+
+CTP payment representation:
```

Comment:
Do you want to add that a transaction is created in this case?

Reply:
Make sense, I will add.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/336/commits/aeb7c7dd508e9d850b6f3d2ed80d281c37d3d67f

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/336/commits/59fe010df330592b41acaecc1d8e2441e6169635